boards/nRF52840-DK/src/lib.rs

```diff
@@ -0,0 +1,580 @@
+//! Board support crate for the Nordic nRF52840-DK
+//!
+//! This crate is in early development. Not much to see here, right now.
+#![no_std]
+
+//#![deny(missing_docs)]
+#![deny(warnings)]
+
+pub use cortex_m;
+pub use cortex_m_rt;
+pub use embedded_hal;
+pub use nrf52840_hal as hal;
+
+/// Exports traits that are usually needed when using this crate
+pub mod prelude {
+    pub use nrf52840_hal::prelude::*;
+}
+
+// TODO:
```

Comment:
Remove TODO

Reply:
I left this for now, but I elaborated on why it is there. Hope that's ok